src/drive/web/modules/upload/index.js

```diff
@@ -85,20 +91,33 @@ export const processNextFile = (
       fileUploadedCallback(uploadedFile)
     }
     dispatch({ type: RECEIVE_UPLOAD_SUCCESS, file })
-  } catch (error) {
-    // eslint-disable-next-line no-console
-    console.warn(error)
-    const statusError = {
-      409: CONFLICT,
-      413: QUOTA
+  } catch (uploadError) {
+    if (uploadError.status === CONFLICT_ERROR) {
+      try {
+        const uploadedFile = await overwriteFile(client, file, dirID)
+        fileUploadedCallback(uploadedFile)
+        dispatch({ type: RECEIVE_UPLOAD_SUCCESS, file, isUpdate: true })
+      } catch (updateError) {
+        error = updateError
+      }
+    } else {
+      error = uploadError
     }
 
-    const status =
-      statusError[error.status] ||
-      (/Failed to fetch$/.exec(error.toString()) && NETWORK) ||
-      FAILED
+    if (error) {
+      // eslint-disable-next-line no-console
+      console.warn(error)
```

Comment:
Nit: I think we started using `minilog`.

Reply:
yep, but it is not merged : https://github.com/cozy/cozy-drive/pull/1662